packages/core/src/components/overlay/_overlay.scss

```diff
@@ -72,19 +72,14 @@ Styleguide components.overlay.scroll
 
 $overlay-background-color: rgba($black, 0.7);
 
-// scroll container for non-inline overlay
-.pt-overlay-scroll-container.pt-overlay-open {
-  @include position(absolute, 0);
-  overflow: auto;
-
-  .pt-overlay-backdrop {
-    // fixed position so it won't scroll under the overlay
-    position: fixed;
-  }
+// restricts scrolling of underlying content while the overlay is open
+body.pt-overlay-open {
+  overflow: hidden;
 }
 
+// fixed position so it won't scroll underneath the overlay content
```

Comment:
not true... fixed position so it always covers the entire viewport (irrespective of page scroll position)